docs/eventing/sources/pingsource.md

```diff
@@ -1,196 +1,261 @@
 ---
-title: "PingSource"
+title: "PingSource example"
 linkTitle: "PingSource"
-weight: 31
+weight: 10
 type: "docs"
+aliases:
+  - ../cronjob-source
 ---
 
-![version](https://img.shields.io/badge/API_Version-v1beta2-red?style=flat-square)
```

Comment:
Not sure why the badge was removed?
@omerbensaadon is there a decision about whether we're keeping these or not? Eng added them afaik.

Reply:
Don't look at me!

Reply:
We may want to update them for v1 though, yeah?

Reply:
pingsource is v1beta2. No v1 yet.

Reply:
@RichardJJG I'd leave these in since eng added them please.

Reply:
Will do (I deleted this by accident anyway, I think)